src/adapter/src/catalog/consistency.rs

```diff
@@ -46,6 +46,7 @@ impl CatalogInconsistencies {
         self.internal_fields.is_empty()
             && self.roles.is_empty()
             && self.comments.is_empty()
+            && self.object_dependencies.is_empty()
             && self.items.is_empty()
```

Comment:
Can we do something like

```suggestion
        let Self {
            internal_fields,
            roles,
            comments,
            object_dependencies
        } = self;
        internal_fields.is_empty()
            && roles.is_empty()
            && comments.is_empty()
            && object_dependencies.is_empty()
            && items.is_empty()
```

to avoid missing fields in the future?

Reply:
Good call, updated!